pennylane/_grad.py

```diff
@@ -97,9 +117,10 @@ def __new__(cls, func, argnum=None, method=None, h=None):
             return ops_loader.grad(func, method=method, h=h, argnums=argnum)
 
         if method or h:  # pragma: no cover
-            raise ValueError(
-                f"Invalid values for 'method={method}' and 'h={h}' in interpreted mode"
-            )
+            raise ValueError(f"Invalid values '{method=}' and '{h=}' without QJIT.")
```

Comment:
Given we want to be able to have catalyst interpret `qml.grad` captured into jaxpr and use that instead, I think we should be able to capture the catalyst keyword arguments as well.

Any necessary validation can occur be whatever consumes the `qml.grad` primitive.

Reply:
Done